app/src/main/res/layout/fragment_geraete_edit.xml

```diff
@@ -0,0 +1,211 @@
+            android:inputType="numberDecimal"
+
+            android:layout_width="wrap_content"
+            android:hint="@string/geraete_edit_vollastHint"
+
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/geraet_edit_editText_margin"
+            app:layout_constraintStart_toStartOf="@+id/geraete_edit_TextVolllast"
+            app:layout_constraintTop_toBottomOf="@+id/geraete_edit_TextVolllast"
+            />
+
+
+        <EditText
+            android:id="@+id/geraete_edit_EditZeit"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="240dp"
+
+            android:layout_marginLeft="240dp"
+
+            android:layout_marginTop="@dimen/geraet_edit_editText_margin"
+            android:inputType="numberDecimal"
+            android:hint="@string/geraete_edit_zeitHint"
+            app:layout_constraintRight_toLeftOf="@+id/geraete_edit_EditVolllast"
+            app:layout_constraintStart_toStartOf="@+id/geraete_edit_TextVolllast"
+            app:layout_constraintTop_toBottomOf="@+id/geraete_edit_TextVolllast" />
+
+        <TextView
+            android:id="@+id/geraete_edit_TextStandBy"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/geraet_edit_textview_margin"
+            android:text="@string/geraet_new_TextStandBy"
+            app:layout_constraintStart_toStartOf="@+id/geraete_edit_EditVolllast"
+            app:layout_constraintTop_toBottomOf="@+id/geraete_edit_EditVolllast" />
+
+        <EditText
+            android:id="@+id/geraete_edit_EditStandBy"
+            android:inputType="numberDecimal"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/geraet_edit_editText_margin"
+            android:hint="@string/geraete_edit_standbyHint"
+
+
+            app:layout_constraintStart_toStartOf="@+id/geraete_edit_TextStandBy"
+            app:layout_constraintTop_toBottomOf="@+id/geraete_edit_TextStandBy" />
+
+        <CheckBox android:id="@+id/geraete_checkbox"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/geraet_edit_textview_margin"
+
+            android:text="@string/geraet_new_TextUrlaubsmodus"
+            app:layout_constraintStart_toStartOf="@+id/geraete_edit_EditStandBy"
+            app:layout_constraintTop_toBottomOf="@+id/geraete_edit_EditStandBy"/>
+
+        <Button
+            android:id="@+id/geraete_edit_save"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="16dp"
+            android:layout_marginTop="35dp"
+            android:layout_marginRight="16dp"
+            android:layout_marginBottom="16dp"
```

Comment:
Margin Top und Bottom sind bei den Buttons ja auch gleich, da wär auch dut, die dann in die dimen auszulagern, so dass wenn man dass bei einem ändert, dann alle dementsprechend verschoben werden.